packages/terminology/src/router.ts

```diff
@@ -0,0 +1,67 @@
+import { Router, Request, Response } from "express";
+import { codeSystemImportHandler } from "./operations/codeImport";
+import { codeSystemLookupHandler } from "./operations/codeLookup";
+import { conceptMapImportHandler } from "./operations/conceptMapImport";
+import { conceptMapTranslateHandler } from "./operations/conceptMapTranslate";
+import { FhirRequest, HttpMethod } from "@medplum/fhir-router";
+import { asyncHandler } from "./util";
+
+const fhirRouter = Router();
+
+function createFhirRequest(req: Request): FhirRequest {
+  return {
+    method: req.method as HttpMethod,
+    pathname: req.originalUrl.replace("/fhir/R4", "").split("?").shift() || "",
+    params: req.params,
+    query: req.query as Record<string, string>,
+    body: req.body,
+    headers: req.headers,
+  };
+}
+
+fhirRouter.post(
+  "/CodeSystem/import",
+  asyncHandler(async (req: Request, res: Response) => {
+    const fhirRequest = createFhirRequest(req);
+    const response = await codeSystemImportHandler(fhirRequest);
+    res.status(200).json(response);
+  })
+);
+
+fhirRouter.post(
+  "/CodeSystem/lookup",
```

Comment:
Are we trying to implement a FHIR API of sorts, with the path using the resource in CamelCase? Our pattern AFAIK is to have those in kebab-case. If we want to have it FHIR compliant, the operation is named with `$` - [source](https://www.hl7.org/fhir/R4/codesystem-operation-lookup.html).

I'd try to simplify and just use our pattern, if being FHIR compliant is not a requirement. Its not as simple as it seems and I think it would be out of scope to be FHIR compliant.